livekit/src/room/mod.rs

```diff
@@ -69,6 +69,8 @@ pub enum RoomError {
     TrackAlreadyPublished,
     #[error("already closed")]
     AlreadyClosed,
+    #[error("request error: {}", .0.message)]
+    Request(proto::RequestResponse),
```

Comment:
Can you recreate another struct, basically having the same fields of the RequestResponse but removing the internal fields like the request_id